bot/exts/moderation/modlog.py

```diff
@@ -522,14 +522,27 @@ async def log_cached_deleted_message(self, message: discord.Message) -> None:
                 f"**Message ID:** `{message.id}`\n"
                 f"**Sent at:** {format_dt(message.created_at)}\n"
                 f"[Jump to message]({message.jump_url})\n"
-                "\n"
             )
 
+        # If the message is a reply, add the reference to the response
+        if message.reference is not None and message.reference.resolved is not None:
+            resolved_message = message.reference.resolved
+
+            if isinstance(resolved_message, discord.Message):
```

Comment:
If the message is deleted (meaning the type is `DeletedReferencedMessage`), can you make it say **In reply to:** *Deleted*

Reply:
You mean to say like, if the parent message get deleted then it will edit the sent messages changing `In reply to: Deleted`?